platform/linux-generic/arch/aarch64/odp_cpu.h

```diff
@@ -59,4 +59,8 @@ do {							     \
 #include "odp_atomic.h"
 #include "odp_cpu_idling.h"
 
+#ifdef __ARM_FEATURE_UNALIGNED
+#define _ODP_UNALIGNED
```

Comment:
This should be defined with value 0 or 1. We try to avoid "#ifdef FOO" and use if(FOO) instead. This way both if and else branches are always compiled and are less likely to break build later on. Optimizer removes the unused branch, so there should not be performance impact.